cinema/models.py

```diff
@@ -102,6 +109,14 @@ def clean(self):
                     }
                 )
 
+    def clean(self):
+        Ticket.validate_seat_and_row(
+            self.seat,
+            self.row,
+            self.movie_session.cinema_hall,
+            ValueError
```

Comment:
The `clean` method uses `ValueError` as the `error_to_raise` argument in `validate_seat_and_row`. It might be more appropriate to use `ValidationError` here, as it is specifically designed for validation issues in Django models.